### PR TITLE
Makes changes from review of PR #1179

### PR DESCRIPTION
**Pull request by JeffH-AWS**
### Description
A few changes were requested in PR #1179 but the approvals were given before the changes were made, and I forgot to push the saved changes I made locally up to GitHub.  This PR just represents the changes.

### Issues Resolved
n/a - follow up PR to 1179

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/remote.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ testClusters {
 }
 ```
 
-Segment replication must also be enabled in order to use remote-backed storage.
+Segment replication must also be enabled to use remote-backed storage.
```
Comment (alicejw-aws) [inline _opensearch/remote.md:32]: must be turned on. . .

**Review by alicejw-aws — APPROVED** [review]
Looks good. . noticed some instances of enable -> turn on, but not sure if this applies or not.

File: _opensearch/remote.md (hunk 1 above)
Comment (JeffH-AWS) [reply _opensearch/remote.md:32]: Use of enable was intentional since it's the same verb used in the property.

Comment (JeffH-AWS) [reply _opensearch/remote.md:32]: Sorry by property I mean:

`systemProperty 'opensearch.experimental.feature.replication_type.enabled', 'true'`

Comment (alicejw-aws) [reply _opensearch/remote.md:32]: oh got it! makes sense. . .thanks

**Review by kolchfa-aws — APPROVED** [review]

**Pull request merged**